model/file/events.yaml

```diff
@@ -0,0 +1,31 @@
+groups:
+  - id: event.file.open
+    stability: experimental
+    type: event
+    name: file.open
+    brief: >
+      A file is defined as a set of information that has been created on,
+      or has existed on a filesystem.
+      A file open event represents the action of a process accessing a file
+      on the filesystem. It includes details such as the file's name,
+      path, directory, size, extension, and metadata, including
+      file access time, file origin information and more. It addition,
+      it also includes information about the process that accessed the file.
+    attributes:
```

Comment:
Hi @lmolkova (CC: @trisch-me, @jsuereth )

Thanks you for the feedback! 

>not OS-wide things
As you pointed out, it was my misunderstanding—I was imagining OS-level events. (Since ECS can also handle that information, I ended up confusing the two)

>let's add note on who/how/when should emit this event.
>E.g. OTel instrumentations are usually run in a certain process

I'm not very familiar with the internal structure of Otel's instrumentation, so to be honest, it's difficult for me to provide an answer at this point. If possible, could you please share the specific technical documentation for the Otel instrumentation you're referring to? Additionally, if OS-level events are not expected, who is expected to send the `file.access` or `file.open` events? Could you share any assumptions or scenarios you had in mind?